setup.py

```diff
@@ -1,48 +1,88 @@
 from __future__ import absolute_import, division, print_function
 
-from setuptools import setup, find_packages, Extension
+import glob
 import os
+import sys
+from setuptools import setup, find_packages, Extension
+from setuptools.command.build_ext import build_ext as _build_ext
 
-import numpy as np
```

Comment:
I'm having issues installing stratify due to the direct use of `numpy` in `setup.py`. This change looks like a valid fix for my error, +1